Source/GSXibInternal.h

```diff
@@ -0,0 +1,42 @@
+/** -*- mode: ObjC -*-
+  <title>GSXibInternal</title>
```

Comment:
I don't see what this class has to do with this pull request. Most likely it belongs into a different one.

Reply:
This class was meant to handle the "embedded" XIBs/models that might result from having cells defined in "prototypeCells" in the XIB file.   For now I think this can be removed since it is no longer needed in this PR as I am leaving it to a future PR since this one is large enough.